### DISCUSSION TITLE
Is it possible to track a RSS feed?

### DISCUSSION THREAD

**Discussion by Caroga** [plausible/analytics#552] [Self-Hosted Support]
Hi there. I just launched a revamp of my blog and I've added a RSS feed into it. I was wondering if it would be possible to track this or not.
I have no clue on how to go about this so any help or feedback would be appreciated, cheers!

**Comment by oliver-kriska** [comment]
Hi, I think it will not work with JS integration. You can call api manually from your code/app where you generate rss feed.

**Reply by Caroga** [reply]
I generate the RSS feed before deploying my app. I was wondering if there would be alternative ways of pinging plausible about this.

**Reply by oliver-kriska** [reply]
you can trigger some api call, you have to check if it's possible to call plausible api without JS, I'm not sure about it. BTW: I'm not plausible developer too :)

**Comment by tcurdt** [comment]
@Caroga Not a plausible dev here but... The only way I see would be by importing your web server logs. Of course this only works if you are not using a CDN. Otherwise you would need to get the data from the CDN. I guess the bottom line is: it's complicated

**Comment by rosano** [comment]
Duplicate of #1356
